wrapper.mjs

```diff
@@ -0,0 +1,5 @@
+import uuid from './dist/index.js';
+export const v1 = uuid.v1;
+export const v3 = uuid.v3;
+export const v4 = uuid.v4;
+export const v5 = uuid.v5;
```

Comment:
Will this work with three-shaking? 🤔 I have a hunch that it doesn't because `uuid` is one object here

Another approach would be something like this:

```suggestion
import uuidv1 from './dist/v1.js';
import uuidv3 from './dist/v3.js';
import uuidv4 from './dist/v4.js';
import uuidv5 from './dist/v5.js';
 export const v1 = uuidv1;
 export const v3 = uuidv3;
 export const v4 = uuidv4;
 export const v5 = uuidv5;
```

Reply:
Wrapper approach is used only for node. Bundlers will get pure esm modules. So treeshaking here is not relevant.